package-res/resources/web/dojo/pentaho/common/TemplatePicker.js

```diff
@@ -83,6 +83,9 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
             },
 
             imgClick: function (event) {
+              if(event.detail > 1){
```

Comment:
@annapopova1 Yes, unfortunately, this solution does not work for the IE. I modified fix and changed styles.